docs/about/compatibility/3rd_party_support_matrix.md

```diff
@@ -1,11 +1,9 @@
-# 3rd Party Support Matrix
+# Third party support matrix
 
 ROCm™ supports various 3rd party libraries and frameworks. Supported versions
 are tested and known to work. Non-supported versions of 3rd parties may also
```

Comment:
Looks good, Lisa. Minor tweak:

- Spell it out? third-party software? Third-party is hyphenated.

Reply:
That sounds great, Lisa. Thanks for letting me know.